aten/src/ATen/templates/UnambiguousTorchOperators.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+// ${generated_comment}
+
+#include <ATen/Functions.h>
+#include <ATen/Tensor.h>
+
+// Extension writers: do you write wrapper functions? Are you frustrated with
+// resolving overloads of operators? Are you frustrated with dealing with
+// pointer-to-methods and resolving overloads of pointer-to-methods?? Look no
+// further, this is the utility for you.
+//
+// Given an operator schema: aten::op.overload(...
+//
+// Use ATEN_FN2(op, overload) to get a *function* version of the operator
+// that is guaranteed to not be overloaded. This means that you can safely
+// decltype(&ATEN_FN2(op, overload)) it. NB: the 2 means this macro takes 2 args.
+//
+// Given an operator schema without an overload name: aten::op(...
+//
+// Use ATEN_FN(op) to get an unambiguous *function* version of the operator.
+//
+// There is some interesting behavior for out= operations.
+// ATEN_FN2(sin, "out") gives a function that is *faithful* to the schema;
```

Comment:
nit: are there supposed to be quotes around `out`?

Reply:
Good catch!